cypress/support/fragments/inventory/inventorySearch.js

```diff
@@ -219,4 +219,8 @@ export default {
   verifySearchResult(cellContent) {
     cy.expect(MultiColumnListCell({ content: cellContent }).exists());
   },
+
+  expandHoldingsSection() {
```

Comment:
this is excess method. Don't use it please.
use openHoldings from InventoryInstance instead

Reply:
thank, done